### PR TITLE
CreateCommentView added.

### PR DESCRIPTION
**Pull request by Erfan2003**
Some Serializer names changed (adding Serializer suffix) really appreciate to follow this convention cause sometimes leads to confusion.

### CODE REVIEW

File: blog/paginators.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from rest_framework.pagination import PageNumberPagination
+
+
+class CommentPaginator(PageNumberPagination):
+    page_size = 20
+    page_size_query_param = 'page_size'
```
Comment (RealNitroZeus) [inline blog/paginators.py:6]: Set page_query_param instead of page size

File: blog/views.py [hunk 2]
```diff
@@ -30,4 +31,13 @@ class ArticleDetailView(DetailView):
 
 class CreateCommentView(CreateAPIView):
     queryset = Comment.objects.all()
-    serializer_class = CreateCommentSerializer
+    serializer_class = CreateComment
+
+
+class GetCommentsView(ListAPIView):
```
Comment (RealNitroZeus) [inline blog/views.py:37]: Remove View at the end of View names

File: blog/views.py [hunk 3]
```diff
@@ -30,4 +31,13 @@ class ArticleDetailView(DetailView):
...
+    serializer_class = GetComment
+    pagination_class = CommentPaginator
+
+    def get_queryset(self):
+        article_id = self.kwargs['pk']
+        return Comment.objects.filter(id=article_id)
```
Comment (RealNitroZeus) [inline blog/views.py:43]: filter it with article__id !

File: blog/paginators.py (hunk 1 above)
Comment (Erfan2003) [reply blog/paginators.py:6]: heeey :(

File: blog/views.py (hunk 2 above)
Comment (Erfan2003) [reply blog/views.py:37]: OK

File: blog/paginators.py (hunk 1 above)
Comment (Erfan2003) [reply blog/paginators.py:6]: Remove the page size param or not?

**Pull request merged**